Analysis/include/Luau/TypePairHash.h

```diff
@@ -3,6 +3,8 @@
 
 #include "Luau/TypeFwd.h"
 
+#include <cstdint>
+
```

Comment:
```suggestion
#include <stdint.h>
```
This is the fix we have internally for the sync next this Friday, would be nice to match it 1:1 to avoid conflicts.

Some background: we use C header because we are not accessing the types like std::uint32_t or std::uintptr_t, we use C typedefs.
C++ headers sometimes define both for compatibility, so it's not a big thing, but it will be consistent with all other includes.